cosmos/appfactory.go

```diff
@@ -28,14 +28,12 @@ type AppFactory struct {
 
 // NewAppFactory returns a new AppFactory.
 func NewAppFactory(logger log.Logger, db dbm.DB) *AppFactory {
-	cdc := codec.New()
-	sdk.RegisterCodec(cdc)
-	codec.RegisterCrypto(cdc)
+	sdk.RegisterCodec(codec.Codec)
+	cosmoscodec.RegisterCrypto(codec.Codec)
```

Comment:
Do you think it should be done directly in codec init function?

Reply:
done